core/base/CMakeLists.txt

```diff
@@ -165,6 +165,7 @@ set(BASE_SOURCES
   src/TRef.cxx
   src/TRegexp.cxx
   src/TRemoteObject.cxx
+  src/TROOT.cxx
```

Comment:
Wasn't this already added to the library in the first commit? Needed here?

Reply:
Nice catch, I think this is a "rebase artifact", this should have been in the first commit indeed. I had to rework my changes against a new version of ROOT because a normal rebase was just not possible. I can rework again to make sure it all works, but it's probably ok to just use a merge commit for the branch instead.

Reply:
I say merge and be done with it, but @Axel-Naumann and @bellenot should have a chance to chime in before merging.